docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,260 @@
+
+### Labels
+The `Passed Manual Tests` label can be added to a PR if you have tested the code on your own VM and determined that it performs as intended.
+
+The `Needs Rebase` label can be added if a PR has merge conflicts with the branch it is based off of.
+
+The `Needs Work` label can be added if a PR is not yet ready for review and requires more work before it can be approved.
+
+Other Labels can also be added by reviewers at their discretion. 
+
+### Code Review Etiquette
+The most important thing to remember during code review is to be respectful and positive in all correspondence with other developers. It is also good practice
+to be on top of all comments or requests and to respond to every comment.
+
+#### When to ask for or remind someone to do a review
+You may ask for someone to review your PR at your discretion. However, do not assign someone to your PR.
+
+In general, if someone has previously reviewed your PR and requested changes, it is good practice to re-request their review once you have implemented those changes.
+
+If you have requested review on a PR and have not received feedback for a number of days, it is okay to either re-request their review or leave a comment
+on the PR pinging them with a short message. For example, "Hey @developer, could you please review this PR and let me know what you think!"
+
+#### Implementing suggestions or requested changes
+It is good practice to implement any requested changes as soon as possible and then re-request review from the developer who requested them. It is also recommended
+that you acknowledge a reviewer's suggestion or request, either by responding to it with a short "I'm on it!"-type message or by leaving an emoji.
+
+If you feel that the suggested or requested change is not necessary, feel free to respond in the comment section with a clear explanation of why you don't think it is necessary.
+Make sure to stay positive and respectful in all review or comment threads!
+
+#### When to resolve a review or comment thread
+If you have implemented a requested change from a review-comment thread, it is good practice to let the developer who requested the change
+resolve the thread. However, if a comment thread is simple or straightforward and has been resolved, feel free to resolve it.
+
+If there is context discussed in a review-comment thread that people may want to consult later, don't resolve the comment thread. You can simply put an emoji and add a comment saying "Thanks" or "Change made" to signal that the review has been addressed.
+
+## 6. Accessing a Database
+To access the database, use the following command, using the credentials given to you:  
+```
+mysql -u user -h host database -p
+```
+and provide the password on prompt.
+
+To preview very large tables, use `\G` instead of `;` at the end of a query. This
+ formats the query output nicely, so that you can read through it easier. 
+
+
```

Comment:
whitespace here should be removed in your next pass to clean up the typos, spacing, indentation, grammar, tone etc...